scripts/build.ps1

```diff
@@ -154,6 +159,10 @@ function CMake-Generate {
             "x64"   { $Arguments += "x64" }
             "arm"   { $Arguments += "arm" }
             "arm64" { $Arguments += "arm64" }
+            "x86_uwp"   { $Arguments += "Win32" }
```

Comment:
nit: can you align all the `{ ... }` above too?

Reply:
Fixed by fixing below